src/core/components/layout/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { HTMLAttributes, ReactNode } from "react"
+import { SerializedStyles } from "@emotion/core"
+import {
+	stack,
+	stackSpace1,
+	stackSpace2,
+	stackSpace3,
+	stackSpace4,
+	stackSpace5,
+} from "./styles"
+import { Props } from "@guardian/src-helpers"
+
+interface StackProps extends HTMLAttributes<HTMLDivElement>, Props {
+	space?: 1 | 2 | 3 | 4 | 5
```

Comment:
would we consider allowing any space in the DS spacing scale?

Reply:
Yeah, I can't think of a reason why not! It would be better for consistency and user expectation. Great suggestion 👍

Reply:
🚀